src/libraries/adaptive-curve/ConstantsLib.sol

```diff
@@ -17,9 +17,9 @@ library ConstantsLib {
     /// @notice Initial rate at target per second (scaled by WAD).
     int256 public constant INITIAL_RATE_AT_TARGET = int256(0.04 ether) / 365 days;
 
-    /// @notice Mininimum rate at target per second (scaled by WAD) (0.1% APR).
-    int256 public constant MIN_RATE_AT_TARGET = int256(0.001 ether) / 365 days;
+    /// @notice Mininimum rate at target per second (scaled by WAD) (0.025% APR).
+    int256 public constant MIN_RATE_AT_TARGET = int256(0.00025 ether) * CURVE_STEEPNESS / 365 days;
```

Comment:
I need to open my eyes...

Reply:
Can you adapt the initial rate at target too ?

Reply:
I wanted at first sight, but it is not obvious why we'd multiply or divide this rate by the curve steepness

We'd multiply it because it is assumed that at market creation, the utilization is 0 so the error is 100%?